### PR TITLE
feat(healthcheck): remove default healthcheck for apps

### PR DESCRIPTION
**Pull request by smothiki**
timeouts and delays defaults to 50 seconds

### CODE REVIEW

**Comment by helgi** [comment]
@smothiki 50 seconds for timeout seems a tad high here. For initial delay it sounds pretty decent, that's what is used as the "warm up cache" type of delay the timeouts will wait from the first time the pod comes up but then the timeout is 50 seconds for hitting `/healthz` or whatever URL is defined.

I understand that timeout would be within the 1 minute connection timeout window but gives me a pause

**Comment by smothiki** [comment]
This PR doesn't fix the failed ruby build pack error . We have to take a deeper look at it.
@helgi I think we should leave setting timeout thing to USER and 50 like close to a min is reasonable.

**Comment by smothiki** [comment]
Let me know if you want me to change delays to a lesser value

**Comment by helgi** [comment]
We'll keep it like this for now at least

File: rootfs/scheduler/__init__.py [hunk 1]
```diff
@@ -652,7 +652,8 @@ def _create_rc(self, name, image, command, **kwargs):  # noqa
             containers[0]["resources"]["limits"]["cpu"] = cpu
 
         # add in healtchecks
-        js_template = self._healthcheck(js_template, **kwargs['healthcheck'])
+        if kwargs.get('healthcheck', {}):
```
Comment (bacongobbler) [inline rootfs/scheduler/__init__.py]: don't we want this to be just `if kwargs.get('healthcheck'):` so it'll fail if the `healthcheck` kwarg doesn't exist?

**Comment by bacongobbler** [comment]
Nice implementation :+1:

File: rootfs/scheduler/__init__.py (hunk 1 above)
Comment (smothiki) [inline rootfs/scheduler/__init__.py]: I'm just extra cautious. Let me know if you want me to change

Comment (bacongobbler) [inline rootfs/scheduler/__init__.py]: yes please :)

**Pull request merged**